Mathlib/Tactic/Sat/FromLRAT.lean

```diff
@@ -436,7 +439,7 @@ partial def buildReify (ctx ctx' proof : Expr) (nvars : Nat) : Expr × Expr := I
   let mut e := e.lowerLooseBVars (nvars+1) (nvars+1)
   let cons := mkApp (mkConst ``List.cons [levelZero]) (mkSort levelZero)
   let nil := mkApp (mkConst ``List.nil [levelZero]) (mkSort levelZero)
-  let rec mkPS depth e
+  let rec /-- Make the expression `ps`. -/ mkPS depth e
```

Comment:
I think we don't actually want this docstring. It's just noise. I think `let rec`s shouldn't require them and we should update the linter instead.